scripts/download_ensembl.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
it has to be removed and other script gets adapted in order to unify ways to generate input files